docs/src/templates/drawerDataGridSample.jsx

```diff
@@ -0,0 +1,166 @@
+import {
+    Button,
+    Drawer,
+    Icon,
+    Input,
+} from 'react-cm-ui';
+import moment from 'moment-timezone';
+import React, { useState } from 'react';
+
+function ModulesDrawerDataGrid() {
+    const [isDrawerOpen, setIsDrawerOpen] = useState(false);
+    const [searchValue] = useState('');
+
+    function onDrawerToggle() {
+        setIsDrawerOpen(!isDrawerOpen);
+    }
+
+    function onClickTest() {
+        console.log('You just clicked the fourth column!'); // eslint-disable-line no-console
+    }
+
+    return (
+        <div>
+            <Button onClick={() => onDrawerToggle()}>Open Drawer</Button>
```

Comment:
Silly question: Can this just be `<Button onClick={onDrawerToggle}>Open Drawer</Button>`?  Or does that not work?  If it does work, we should update other placements.  I'd say NAB since it's an issue in the Docs app, and not in actual components.